webapp/channels/src/components/sidebar/sidebar_category/sidebar_category.tsx

```diff
@@ -75,38 +74,6 @@ export default class SidebarCategory extends React.PureComponent<Props, State> {
         }
     }
 
-    componentDidMount() {
-        this.categoryTitleRef.current?.addEventListener(A11yCustomEventTypes.ACTIVATE, this.handleA11yActivateEvent);
-        this.categoryTitleRef.current?.addEventListener(A11yCustomEventTypes.DEACTIVATE, this.handleA11yDeactivateEvent);
-    }
-
-    componentWillUnmount() {
-        this.categoryTitleRef.current?.removeEventListener(A11yCustomEventTypes.ACTIVATE, this.handleA11yActivateEvent);
-        this.categoryTitleRef.current?.removeEventListener(A11yCustomEventTypes.DEACTIVATE, this.handleA11yDeactivateEvent);
-
-        if (this.a11yKeyDownRegistered) {
-            this.handleA11yDeactivateEvent();
-        }
-    }
-
-    handleA11yActivateEvent = () => {
-        this.categoryTitleRef.current?.addEventListener('keydown', this.handleA11yKeyDown);
-
-        this.a11yKeyDownRegistered = true;
-    };
-
-    handleA11yDeactivateEvent = () => {
-        this.categoryTitleRef.current?.removeEventListener('keydown', this.handleA11yKeyDown);
-
-        this.a11yKeyDownRegistered = false;
-    };
-
-    handleA11yKeyDown = (e: KeyboardEvent<HTMLButtonElement>['nativeEvent']) => {
-        if (isKeyPressed(e, Constants.KeyCodes.ENTER)) {
-            this.handleCollapse();
-        }
-    };
-
```

Comment:
this is a bit concerning, why did we had to remove all this

Reply:
Agreed, I didn't like the idea of removing it. However I did so because from what I could tell these listeners were being set up to capture ENTER key presses.

But from my testing such listeners aren't required to trigger the collapse/expand functionality, and in fact the existence of these listeners causes it so when you click ENTER on a channel category it neither collapses or expands because it activates `handleCollapse` twice in a row.

Reply:
I should have added in here, if these are in fact required for something that I missed please let me know so I can work on finding a different solution to the issue

Reply:
@M-ZubairAhmed, What do you think about his comment?

Reply:
@M-ZubairAhmed Gentle ping

Reply:
Ah apologies on missing this out. 

> this.categoryTitleRef.current?.addEventListener(A11yCustomEventTypes.ACTIVATE, this.handleA11yActivateEvent);

Dont we still need this though?